main/src/main/java/cgeo/geocaching/settings/fragments/PreferenceMapSourcesFragment.java

```diff
@@ -33,11 +35,14 @@ public void onCreatePreferences(final Bundle savedInstanceState, final String ro
 
         initMapSourcePreference();
 
-        final boolean showUnifiedMap = Settings.getBoolean(R.string.pref_showUnifiedMap, false);
+        final boolean showUnifiedMap = Settings.getBoolean(R.string.pref_showUnifiedMap, false) || !BranchDetectionHelper.isProductionBuild();
```

Comment:
I would advise to just call `Settings.useUnifiedMap` here (instead of repeating the logic expression used there). This would keep the "decision logic" whether to use unified map or not in one place.

Reply:
This line is for deciding whether the unified map section is visible inside the settings.

If we make it depend on `useUnifiedMap` a nightly user could disable unified map easily but without the possibility to re-enable it.